### ISSUE TITLE
Not working in IE11 without polyfill

### ISSUE THREAD

**Issue by digital88** [glidejs/glide#625]
Object.assign have to be polyfilled before glide starts working. Should have noted this in docs.

**Comment by martin-badin** [comment]
I don't think so. Microsoft stopped maintaining IE11 and currently, it is deprecated. It is fine if packages like this stop supporting it.

**Comment by digital88** [comment]
> I don't think so. Microsoft stopped maintaining IE11 and currently, it is deprecated. It is fine if packages like this stop supporting it.

Well it has no mention of IE11 being unsupported in docs. So at least having supported browsers section in docs would be nice.
